packages/node/src/lib/express-middleware.ts

```diff
@@ -100,6 +146,15 @@ export function expressMiddleware(readmeApiKey: string, group: GroupingFunction,
       // this is fine for now
       const groupData = group(req, res);
 
+      let requestBody = req.body;
+      if (req.is('text/*')) {
```

Comment:
Manually bring over is() function: https://github.com/expressjs/express/blob/master/lib/request.js#L252-L290